synapse/replication/http/presence.py

```diff
@@ -114,7 +114,6 @@ async def _handle_request(  # type: ignore[override]
         await self._presence_handler.set_state(
             UserID.from_string(user_id),
             content["state"],
-            content["ignore_status_msg"],
```

Comment:
Can `ignore_status_msg` be removed from `_serialize_payload` above as well?

Reply:
Yeah, I would assume so

Reply:
Can we remove the parameter too? I believe it was only used in the call to `_set_state_client()` in `WorkerPresenceHandler.set_state()`, which has already been updated.

Reply:
ah, good spot - done